lib/index.js

```diff
@@ -7,6 +7,10 @@ var SphericalMercator = require('sphericalmercator');
 var Caching = require('caching');
 var clone = require('clone');
 var connectTimeout = require('connect-timeout');
+var mapnik = require('mapnik');
```

Comment:
@devTristan @lkazberova does this have a memory / startup cost? should loading mvt support be a startup option?